suites/quincy/rados/tier-2_rados_test_mclock_scheduler.yaml

```diff
@@ -0,0 +1,115 @@
+# Suite contains basic tier-2 rados tests
```

Comment:
Let's update more about test suite in the comment.
- config
- scenario

Reply:
Implemented.